packages/devtools_app/lib/src/shared/config_specific/drag_and_drop/drag_and_drop.dart

```diff
@@ -19,7 +19,7 @@ abstract class DragAndDropManager {
 
   static DragAndDropManager get instance => _instance;
 
-  static late final DragAndDropManager _instance = DragAndDropManager();
```

Comment:
why remove late? this will have implications of the `init` method getting called sooner than it would have previously

Reply:
Created issue: https://github.com/dart-lang/linter/issues/4307

Reply:
It turned out static fields are lazy anyway.